net-dns/maradns/maradns-3.5.0020.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+inherit python-any-r1 systemd toolchain-funcs
+
+DESCRIPTION="A security-aware DNS server"
+HOMEPAGE="https://maradns.samiam.org"
+SRC_URI="https://maradns.samiam.org/download/${PV%.*}/${PV}/${P}.tar.xz"
+
+# The GPL-2 covers the init script, bug 426018.
+LICENSE="BSD-2 GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="examples ipv6"
+
+BDEPEND="${PYTHON_DEPS}
+	dev-lang/perl"
+DEPEND="
+	acct-group/maradns
+	acct-user/duende
+	acct-user/maradns"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-flags-ipv6.patch
+)
+
+src_prepare() {
+	default
+	python_fix_shebang tools/bind2csv2.py
+}
+
+src_configure() {
+	tc-export CC
+	./configure \
+		$(usex ipv6 --ipv6 "") || die "Failed to configure"
```

Comment:
Question, why wouldn't `econf` work here?

Reply:
Because author's configure incompatible by flags. It accept only --ipv6 or nothing.